langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/LangLibSubTypeTest.java

```diff
@@ -273,7 +274,7 @@ public void testNegativeIntSubType() {
         BAssertUtil.validateError(result, err++, "incompatible types: expected 'int:Unsigned16', found 'int'", 352, 25);
         BAssertUtil.validateError(result, err++, "incompatible types: expected 'int:Unsigned16', found 'int'", 353, 25);
         BAssertUtil.validateError(result, err++, "incompatible types: expected 'int:Unsigned8', found 'int'", 354, 24);
-        BAssertUtil.validateError(result, err++, "incompatible types: expected 'int:Unsigned8', found 'int'", 355, 24);
+        BAssertUtil.validateError(result, err++, "'9223372036854775808' is out of range for 'Integer'", 355, 25);
```

Comment:
Why are we saying `Integer`? How does this differentiate between `int` and `int` subtypes. Please use the relevant type, instead of an arbitrary name.